From 5d0f8b175f98dc1d622f000442316c7a4aa727e0 Mon Sep 17 00:00:00 2001
Message-Id: <5d0f8b175f98dc1d622f000442316c7a4aa727e0.1502135873.git.jan.steffens@gmail.com>
In-Reply-To: <266ddc3e38107862811927371cbf4b88fdfe4872.1502135873.git.jan.steffens@gmail.com>
References: <266ddc3e38107862811927371cbf4b88fdfe4872.1502135873.git.jan.steffens@gmail.com>
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Mon, 7 Aug 2017 21:39:49 +0200
Subject: [PATCH 3/3] meson: Use the right dependency for gtkdoc

The gdkpixbuf_dep pulls in the right include dir. This fixes the
following error:

    gdk-pixbuf-scan.c:7:10: fatal error: gdk-pixbuf/gdk-pixbuf.h: No such file or directory
---
 docs/reference/gdk-pixbuf/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/reference/gdk-pixbuf/meson.build b/docs/reference/gdk-pixbuf/meson.build
index 5d7c9b6c1cc26579..5d4bae492fb5d037 100644
--- a/docs/reference/gdk-pixbuf/meson.build
+++ b/docs/reference/gdk-pixbuf/meson.build
@@ -31,7 +31,7 @@ if get_option('with_docs')
                  gdk_pixbuf_inc,
                  gdkpixbuf_xlib_inc,
                ],
-               dependencies: gdk_pixbuf_deps,
+               dependencies: gdkpixbuf_dep,
                gobject_typesfile: 'gdk-pixbuf.types',
                scan_args: [
                  '--deprecated-guards="GDK_PIXBUF_ENABLE_BROKEN|GDK_PIXBUF_DISABLE_DEPRECATED"',
-- 
2.13.2

